core/src/main/resources/lib/hudson/project/makeDisabled.jelly

```diff
@@ -41,14 +41,5 @@ THE SOFTWARE.
         </form>
       </div>
     </j:when>
-    <j:otherwise>
```

Comment:
should we also remove the enable button? or is it helpful in the rare case a job is disabled?

Reply:
I think it's fine to stay as its not taking up any more space than its parent banner.